inc/toolbox.class.php

```diff
@@ -252,6 +252,7 @@ public static function getGlpiItemtypes(): array
         ];
 
         $tools_itemtypes = [
+            KnowbaseItem::class,
```

Comment:
Could you please move it after RSSFeed ? I tried to use the same sections/order as in main menu.